docs/source/quick-start.mdx

```diff
@@ -0,0 +1,96 @@
+# Quick start
+
+The Hugging Face Hub is the go-to place for sharing machine learning models, datasets, and metrics. To help you get the most out of the Hub, we created the `huggingface_hub` library so you can interact with the Hub without leaving your development environment. You can create and manage repositories easily and download and upload files from the Hub. 
+
+## Install the Hub library
+
+To get started, you should install the `huggingface_hub` library. The library provides functions that make interacting with the Hub simple. Run the following command to install the library:
+
+```bash
+>>> pip install huggingface_hub
+```
+
+## Download files from the Hub
+
+Repositories store versioned model and dataset files anyone can download. You can use the [`hf_hub_url`] function to download files. This function will download and cache a file on your local disk. The next time you need that file, it will load from your cache, so you don't need to re-download it.
+
+You will need the repository id and the filename of the file you want to download. For example, to download the [Pegasus](https://huggingface.co/google/pegasus-xsum) model configuration file: 
+
+```py
+>>> from huggingface_hub import hf_hub_download
+>>> hf_hub_download(repo_id="google/pegasus-xsum", filename="config.json")
+```
+
+To download a specific version of the file, use the `revision` parameter to specify the branch name, tag, or commit hash. If you choose to use the commit hash, it must be the full-length hash instead of the shorter 7-character commit hash: 
+
+```py
+>>> from huggingface_hub import hf_hub_download
+>>> hf_hub_download(repo_id="google/pegasus-xsum", filename="config.json", revision="4d33b01d79672f27f001f6abade33f22d993b151")
+```
+
+Finally, you can choose where to cache a file with the `cache_dir` parameter:
+
+```py
+>>> from huggingface_hub import hf_hub_download
+>>> hf_hub_download(repo_id="google/pegasus-xsum", filename="config.json", cache_dir="/my/cool/cache")
+```
+
+## Create a repository
+
+To create and share files to the Hub, you need to have a Hugging Face account. [Create an account]((https://huggingface.co/join)) if you don't already have one, and then sign in to find your [User Access Token](https://huggingface.co/docs/hub/security#user-access-tokens) in your Settings. The User Access Token is used to authenticate your identity to the Hub.
+
+1. Log in to your Hugging Face account with the following command:
+
+```bash
+huggingface-cli login
+```
+
+Or if you prefer to work from a Jupyter or Colaboratory notebook, then login with [`notebook_login`]:
+
+```py
+>>> from huggingface_hub import notebook_login
+>>> notebook_login()
+```
+
+2. Enter your User Access Token to authenticate your identity to the Hub.
+
+After you've created an account, create a repository with the [`create_repo`] function:
+
+```py
+>>> from huggingface_hub import create_repo
+>>> create_repo(repo_name="super-cool-model")
+```
```

Comment:
I rather tell users to use `HfApi().create_repo` rather than the provided method.

Reply:
So do this instead (and the same for below)?

```py
from huggingface_hub import HfApi

api = HfApi()
api.create_repo(repo_id="super-cool-model")
```

Reply:
I think there was a strong push from folks to use `create_repo` instead of `HfApi()` due to confusions some time ago.

Reply:
`create_repo` _is_ `HfApi()`, it's just a shorthand for the default instance of `HfApi()`. The API would look much nicer once we store the token in the instance rather than somewhere on disk.